### DISCUSSION TITLE
I want add methods like getAllSheetNames() and to get particular sheet by getSheet(index) method.

### DISCUSSION THREAD

**Discussion by Firewolf-10** [pjfanning/excel-streaming-reader#182] [Ideas]
Would it be appropriate for me to raise the above issues as a proposal for improving the user experience through the addition of new methods?
Can I work on it?

**Comment by pjfanning** [comment]
I'll review any PRs but I'd prefer not to implement custom APIs (ie adding methods that are not part of the POI SS interfaces).

**Reply by Firewolf-10** [reply]
yes. I agree but there are still methods that need to be implemented to use this dependency in projects. otherwise, looping every time for developers will be a little difficult and it will consume more time.

**Reply by pjfanning** [reply]
getSheetAt(int) is an existing API on the workbook
